gatsby-config.js

```diff
@@ -37,6 +46,22 @@ module.exports = {
         },
         `gatsby-transformer-sharp`,
         `gatsby-plugin-sharp`,
+        {
+            resolve: `gatsby-plugin-intl`,
+            options: {
+                path: `${__dirname}/src/translations`,
+                languages: [`en`, `es`],
+                defaultLanguage: `en`,
+                redirect: true,
+            },
+        },
+        {
+          resolve: `gatsby-source-contentful`,
+          options: {
+            spaceId: `tqd0xcamk1ij`,
+            accessToken: process.env.CONTENTFUL_ACCESS_TOKEN,
```

Comment:
First time we're introducing environment config files to store Contentful access tokens